tests/cross-language/scripts/decrypt_all.sh

```diff
@@ -43,7 +43,7 @@ export ASHERAH_EXPIRE_AFTER=90
 export ASHERAH_CHECK_INTERVAL=10
 export ASHERAH_METASTORE_MODE=jdbc
 export ASHERAH_CONNECTION_STRING='jdbc:mysql://127.0.0.1:3306/testdb?user=root&password=Password123'
-java -jar /home/circleci/.m2/repository/com/godaddy/asherah/grpc-server/1.0.0-SNAPSHOT/*dependencies.jar &
+java -jar ~/.m2/repository/com/godaddy/asherah/grpc-server/1.0.0-SNAPSHOT/*dependencies.jar &
```

Comment:
this looks invalid. It seems that we are looking for a jar in `~/.m2`. There wouldn't be an updated jar in the cache since you removed the `restore java cache` option

Reply:
can discuss the different approaches offline on how to tackle this

Reply:
the jar will be pushed to ~/.m2 every time we build the Java CLTF https://github.com/sushantmimani/asherah/blob/62f97205bc69157313c53db8bcdce5d1b8722f37/.circleci/config.yml#L685

Reply:
discussed an approach where we get rid of local install completely. This makes sure that we are always using the build from the current branch/tag.